### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by SeuRonao** [NateHawk85/Nathans-DnD5e-Foundry-Modules#1]
I believe these modules should be considered for reviewing:

[PNP - Pointers and Pings](https://foundryvtt.com/packages/pointer/), to substitute pings
[Automatic Journal Icon Numbers](https://foundryvtt.com/packages/journal-icon-numbers/), makes having a multi room dungeon more manageable.
[Token Tooltip Alt](https://foundryvtt.com/packages/token-tooltip-alt/), all the tooltip things are almost possible.

**Comment by NateHawk85** [comment]
Hey! Sorry for the late response.

I took a look at the modules you linked. For PNP, that's actually one that I debated back and forth on before finalizing Pings as our personal solution. We simply didn't really need the additional functionality of PNP, and the "long-press to ping" seemed more intuitive to us.

As for Automatic Journal Icon Numbers, that could potentially be something to look at. I'm currently a fan of Pin Cushion, and I'm concerned how nicely this module would play with it. But I will add it to my "Modules to Review" section, as it could be a good QoL addition.

As for Token Tooltip Alt, I wonder how that plays with some of the modules I already have installed that affect tooltips for tokens. The customization is nice, but might take some tweaking. Will also add this to my "Modules to Review" section!

Thanks for your suggestions!
